### DISCUSSION TITLE
Adopt "moveableRedstoneStuff" from suscarpetaddons

### DISCUSSION THREAD

**Discussion by ghost** [Minecraft-AMS/Carpet-AMS-Addition#114] [Ideas]
https://github.com/11-90-an/suscarpetaddons

Interesting rule "**movableRedstoneStuff**".
Redstone dust, redstone torches, repeaters and comparators can be moved now.

Maybe splitting it into 4 different rules.
- moveable RedstoneDust
- moveable RedstoneTorch
- moveableRepeater
- moveableComparator

**Comment by wendavid552** [comment]
That's interesting, btw moving these redstone stuff is probably inconsistent with some optimization mods e.g. Lithium. I'm not sure if we could keep our compatability in most cases.

Some of the redstone stuff e.g. redstone dust, if movable, may cause weird problem in update orders.
